README.md

```diff
@@ -1,6 +1,32 @@
-![PySnip](http://i.imgur.com/QFgqcRM.png)
 
-[![Build Status](https://travis-ci.org/NateShoffner/PySnip.svg?branch=master)](https://travis-ci.org/NateShoffner/PySnip)
+# About this fork #
+
+[![Build Status](https://travis-ci.org/swalladge/PySnip.svg?branch=master)](https://travis-ci.org/swalladge/PySnip)
+
+This fork (original project at [NateSchoffner/PySnip](https://github.com/NateShoffner/PySnip)) has been created with the aim to complete the following:
+
+- [DONE] separate server from configs/maps/scripts/etc (at the moment it's all combined, and messy)
+- make it installable via pip or a linux package manager
+- port to python3
+- keep up to date and fix any bugs (please send pull requests!)
+- provide some more documentation on how to instal, run, and configure the server
+
+
+## FAQs

```

Comment:
This note is a useful one to keep around - whether you want to merge it in with a FAQ in your readme, or start a wiki?